src/widgets/search-box/search-box.js

```diff
@@ -173,8 +207,9 @@ searchBox({
  * @property  {string|HTMLElement} container CSS Selector or HTMLElement to insert the widget.
  * @property  {string} [placeholder] Input's placeholder.
  * @property  {boolean|SearchBoxPoweredByOption} [poweredBy=false] Define if a "powered by Algolia" link should be added near the input.
- * @property  {boolean|SearchBoxResetOption} [reset=true] Define if a reset button should be added in the input when there is a query.
+ * @property  {boolean|SearchBoxResetOption} [reset=false] Define if a reset button should be added in the input when there is a query.
```

Comment:
Is it false by default now to have a reset button?

Reply:
Will double check 👍